samples/SoundBoard/src/main/java/com/google/oboe/samples/soundboard/MusicTileView.java

```diff
@@ -134,7 +127,14 @@ public boolean onTouchEvent(MotionEvent event) {
 
         // Calling invalidate() will force onDraw() to be called
         if (didImageChange) {
-            isInitialOnDrawCall = false;
+            // Call c++ native code to set each index add/remove sound
+            for (int i = 0; i < mRectangles.size(); i++) {
+                if (mIsPressedPerRectangle[i]) {
+                    setNoteOn(mEngineHandle, i);
```

Comment:
This could cause multiple noteOn() events if a finger is held down while another finger is moved.

Reply:
Thanks for bringing this up! I have changed the logic when the finger has moved